src/interpreter_funcs.rs

```diff
@@ -394,6 +404,62 @@ impl Func for FuncImplCreatePlane {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub enum FuncImportObjMeshError {
+    Empty,
+    Importer(ImporterError),
+}
+
+impl fmt::Display for FuncImportObjMeshError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            Self::Empty => write!(f, "No mesh geometry contained in OBJ"),
+            Self::Importer(importer_error) => f.write_str(&importer_error.to_string()),
+        }
+    }
+}
+
+impl error::Error for FuncImportObjMeshError {}
+
+pub struct FuncImplImportObjMesh<C: ObjCache> {
+    importer: Importer<C>,
+}
+
+impl<C: ObjCache> Func for FuncImplImportObjMesh<C> {
+    fn flags(&self) -> FuncFlags {
+        FuncFlags::empty()
+    }
+
+    fn param_info(&self) -> &[ParamInfo] {
+        &[ParamInfo {
+            ty: Ty::String,
+            optional: false,
+        }]
+    }
+
+    fn return_ty(&self) -> Ty {
+        Ty::Geometry
+    }
+
+    fn call(&mut self, values: &[Value]) -> Result<Value, FuncError> {
+        let path = values[0].unwrap_string();
+
+        let result = self.importer.import_obj(path);
+        match result {
+            Ok(models) => {
+                // FIXME: @Correctness Return all geometries once we support all of them
```

Comment:
Rather `Join all geometries into one mesh`. We need to implement such function first (it's pretty high in the backlog), so this is good enough for now.